setup.py

```diff
@@ -63,25 +67,28 @@ def finalize_options(self):
         if self.install_dir is None:
             installobj = self.distribution.get_command_obj('install')
             self.install_dir = installobj.install_lib
-        print 'Installing data files to %s' % self.install_dir
+        print('Installing data files to %s' % self.install_dir)
         install_data.install_data.finalize_options(self)
 
+
 # clean up xpa as well
 class my_clean(clean.clean):
     def run(self):
         make('clean')
         clean.clean.run(self)
 
+
 # setup command
 setup(name='pyds9',
-    version='1.7',
-    description='Python/DS9 connection via XPA (with numpy and pyfits support)',
-    author='Bill Joye and Eric Mandel',
-    author_email='saord@cfa.harvard.edu',
-    url='http://hea-www.harvard.edu/saord/ds9/',
-    py_modules=['pyds9', 'xpa', 'ds9'],
-    data_files=[('', [xpadir+'/'+xpalib, xpadir+'/'+xpans])],
-    cmdclass = {'build_py': my_build_py, 	 \
-                'install_data': my_install_data, \
-                'clean': my_clean },
-   )
+      version='1.7',
+      description='Python/DS9 connection via XPA (with numpy and pyfits support)',
+      author='Bill Joye and Eric Mandel',
+      author_email='saord@cfa.harvard.edu',
+      url='http://hea-www.harvard.edu/saord/ds9/',
+      py_modules=['pyds9', 'xpa', 'ds9'],
+      data_files=[('', [xpadir+'/'+xpalib, xpadir+'/'+xpans])],
```

Comment:
Does this work on Windows?
I think you have to use `os.path.join`, no?

I assume Windows worked at some point, because "Windows" and "CYGWIN" are mentioned in some parts of the code?